cfme/configure/access_control.py

```diff
@@ -232,17 +257,65 @@ def copy(self):
         assert view.is_displayed
         return new_user
 
-    def delete(self, cancel=True):
-        """ Delete existing user
-            Args:
-                cancel: Default value 'True', user will be deleted
-                        'False' - deletion of user will be canceled
+    def _delete_using_all_selection(self):
         """
-        view = navigate_to(self, 'Details')
-        view.configuration.item_select('Delete this User', handle_alert=cancel)
+        Delete existing user by selecting it from the access control group list
+
+        Throws:
+            RBACOperationBlocked: If operation is blocked due to current user
```

Comment:
Good catch. That docstring was a unnecessary leftover from when the function would completely delete the user and throw RBACOperationBlocked. That part of the function was removed since it was duplicate functionality with the normal process of deleleting using User Details